service/src/test/java/uk/nhs/adaptors/gp2gp/common/storageconnectors/LocalMockConnectorTest.java

```diff
@@ -0,0 +1,52 @@
+package uk.nhs.adaptors.gp2gp.common.storageconnectors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import org.apache.commons.io.FileUtils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class LocalMockConnectorTest {
+
+    private static StorageConnector storageConnector;
+    private static final String FILE_TO_UPLOAD = "src/test/resources/test.txt";
+    private static final String FILE_EXPECTED_DOWNLOAD = "src/test/resources/downloads/test.txt";
+    private static final String SETUP_FILE = "setupFile.txt";
```

Comment:
@mmackiewicz1 & @Broom94 create const class for strings reused

Reply:
Strings no longer being reused mock test class removed